### PR TITLE
Feature 5 flashcard creator and remover tests errors

### PR DESCRIPTION
**Pull request by Junyu11111**
My tests are showing errors which stopped the program from running. I will work on it soon. Meanwhile, just remove them from the program for now.

### CODE REVIEW

**Review by ThomasShimUoft — APPROVED** [review]
Alright, that's a good quick temporary fix.

**Comment by Junyu11111** [comment]
Tests for flashcard creation and removal were implemented.
Bug fix in DBGateway.
Changed flashcard term and definition input form.
Minor documentaion changes.

**Review by ThomasShimUoft — APPROVED** [review]
The documentation looks good and the little fixes seem pretty good as well.

File: src/main/java/create_flashcard_use_case/fcCScreens/FcCResponsePresenter.java [hunk 1]
```diff
@@ -3,6 +3,11 @@
 import create_flashcard_use_case.FcCOutputBoundary;
 import create_flashcard_use_case.FcCResponseModel;
 
+/**
```
Comment (Edward-1117) [inline src/main/java/create_flashcard_use_case/fcCScreens/FcCResponsePresenter.java:6]: Good Javadoc for presenter class

File: src/test/java/create_flashcard_use_case/FcCInteractorTest.java [hunk 2]
```diff
@@ -11,8 +11,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
```
Comment (Edward-1117) [inline src/test/java/create_flashcard_use_case/FcCInteractorTest.java:14]: Good Javadoc for testing use case.

**Review by Edward-1117 — APPROVED** [review]
Good job including the Javadocs for your classes and test cases!

**Pull request merged**